### DISCUSSION TITLE
Supress a level from list.html

### DISCUSSION THREAD

**Discussion by technosprout** [thuliteio/doks#1070] [Q&A]
I have a docs setup as

content>en>guides
... _index.md

content>en>guides>help>
... _index.md
... faq.md
... how_to_update.md
... troubleshooting.md

content>en>guides>prologue>
... _index.md
... commands.md
... introduction.md
... quickstart.md
 

mysite/guides/
displays 2 cards for help and prologue (works)

When the user navigates to mysite/guides/prologue

I want to to redirect to the top weighted page 
mysite/guides/prologue to mysite/guides/prologue/commands/

instead of 
mysite/guides/prologue 
displaying 3 cards commands; introduction, quickstart

for the layout list.html
it seems to function the same at every directory level

 {{ $paginator := .Paginate ( where .Pages  "Section" .Section) }}
      {{ range $paginator.Pages }}
         {{ partial "main/docs-card.html" . }}
     {{ end }}

**Comment by h-enk** [comment]
Not quite sure if this can be achieved with Hugo properly (I don't think so). Please ask in the [Hugo Discourse](https://discourse.gohugo.io/) (and share your findings here).

I think the proper way would be using redirects on server level. When e.g. using Netlify, see [Redirects and rewrites](https://docs.netlify.com/routing/redirects/)
